donors.json

```diff
@@ -0,0 +1,1186 @@
+[
```

Comment:
Are all these names of the current sponsors? Are they all visible and agreed to have their names listed with the amount? If not, we should remove this file completely from the repo and add it through Django admin.